crates/runtime/src/instance/allocator/pooling.rs

```diff
@@ -234,21 +233,12 @@ impl Default for ModuleLimits {
 pub struct InstanceLimits {
     /// The maximum number of concurrent instances supported.
     pub count: u32,
-
-    /// The maximum size, in bytes, of host address space to reserve for each linear memory of an instance.
-    pub memory_reservation_size: u64,
 }
 
 impl Default for InstanceLimits {
     fn default() -> Self {
         // See doc comments for `wasmtime::InstanceLimits` for these default values
-        Self {
-            count: 1000,
-            #[cfg(target_pointer_width = "32")]
-            memory_reservation_size: 10 * (1 << 20), // 10 MiB,
```

Comment:
This default value of 1 GiB for 32-bit hosts was the reason the pooling allocator didn't use `static_memory_bound`, as there isn't enough address space for a default of 1000 instances (and `2` seems like a really silly default for the count as an alternative).

Thoughts on this?

Reply:
AFAIK wasmtime doesn't actually even run on any 32-bit platforms right now, or at least we definitely don't test it. Any objection to changing the default in `Tunables` to 10MB like it is here? Only being able to have 2 instances is also a little silly

Reply:
I have no objections; worth it certainly if it removes a duplicate configuration point.